### ISSUE TITLE
[documentation] add explanation why --no-quarantine is required

### ISSUE THREAD

**Issue by itsamemarkus** [sbarex/SourceCodeSyntaxHighlight#271]
For https://github.com/sbarex/SourceCodeSyntaxHighlight/blob/master/README.md

**Comment by MPV** [comment]
>Without this users will get a malicious software warning

...is the only reason I've yet to find (in the PR adding it):
- https://github.com/sbarex/SourceCodeSyntaxHighlight/pull/140

**Comment by MPV** [comment]
As described upstream by @foxt

> The application is (incorrectly), ad-hoc signed. Ad-hoc signing generates a unique signature that will only work for that machine, the codesignature will be classed as invalid on everyone elses machine requiring the quarantine flag removing from the application

**Comment by jtsternberg** [comment]
so, are we dead in the water with this app?

**Comment by sbarex** [comment]
> so, are we dead in the water with this app?

No, The app continue to works. You can always download the app from this repo. And the app can autoupdate.
In this year will be released a signed version.
Please see #328.
